rllib/utils/actor_manager.py

```diff
@@ -232,8 +232,8 @@ class _ActorState:
     def __init__(
         self,
         actors: Optional[List[ActorHandle]] = None,
-        max_remote_requests_in_flight_per_actor: Optional[int] = 2,
```

Comment:
These are not optional. They must be ints. Yes, they have default (int) values, but cannot be None.